airbyte-integrations/connectors/destination-mssql-v2/src/test-integration/kotlin/io/airbyte/integrations/destination/mssql/v2/MSSQLContainerHelper.kt

```diff
@@ -63,4 +63,7 @@ class MSSQLConfigUpdater : ConfigurationUpdater {
         logger.debug { "Using updated MSSQL configuration: $updatedConfig" }
         return updatedConfig
     }
+
+    override fun setDefaultNamespace(config: String, defaultNamespace: String): String =
```

Comment:
I didn't exactly follow where is this default namespace coming from?

Reply:
it's this call https://github.com/airbytehq/airbyte/blob/a225b78b9aa249c5f761c0571344b5f66f5f4357/airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt#L471